lib/compiler/compileDocker.ts

```diff
@@ -1,16 +1,16 @@
 import {join} from 'path';
-import ImportMappingBuilder from './importMappingBuilder';
 import {Config} from '../config/config';
 import {execSync} from 'child_process';
-import {buildInputObject, buildSources} from './buildUitls';
+import {buildInputObject, buildSourcesObject} from './buildUitls';
+import { ImportFile } from '@resolver-engine/imports';
 
 const CONTAINER_PATH = '/home/project';
 const NPM_PATH = '/home/npm';
 
 export function compileDocker(config: Config) {
-  return async function compile(sources: string[]) {
+  return async function compile(sources: ImportFile[]) {
     const command = createBuildCommand(config);
-    const input = JSON.stringify(buildInputJson(sources, config), null, 2);
+    const input = JSON.stringify(buildInputObject(buildSourcesObject(sources)), null, 2);
```

Comment:
This could be simplified to just `buildInputObject(sources)` by having `buildInputObject` call `buildSourcesObject`